### PR TITLE
ref(ui) Convert Event Errors to a styled component

### PR DESCRIPTION
**Pull request by markstory**
We want to use the `EventEntries` component in discover 2 so that when a new interface is added to events it shows up in both issues and discover with only one set of updates.

Unfortunately many parts of issue details are coupled to the specific class hierarchy on that page. This moves event errors away from less and helps make using EventEntries more portable.

### CODE REVIEW

File: src/sentry/static/sentry/app/components/events/errors.jsx [hunk 1]
```diff
@@ -44,26 +44,78 @@ class EventErrors extends React.Component {
...
           })}
-        </ul>
-      </EventDataSection>
+        </ErrorList>
+      </Section>
     );
   }
 }
 
+// TODO don't use a custom pink
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/events/errors.jsx]: maybe `TODO(theme)`?

**Review by billyvg — COMMENTED** [review]
Via percy, it looks like there's a border color that's not right with the change and a failing acceptance test.

Shared issue view seems fine, which would have been my only other concern

File: src/sentry/static/sentry/app/components/events/errors.jsx [hunk 2]
```diff
@@ -44,26 +44,84 @@ class EventErrors extends React.Component {
...
     const isOpen = this.state.isOpen;
     return (
-      <EventDataSection event={this.props.event} type="errors" className="errors">
-        <span className="icon icon-alert" />
-        <p>
-          <a className="pull-right errors-toggle" onClick={this.toggle}>
+      <Section>
+        <Summary>
+          <span>
+            <StyledIconWarning color={theme.red} />
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/events/errors.jsx]: Wouldn't it be better to use the theme inside the `StyledIconWarning` to change the color? then you could get rid of the theme import.

**Comment by priscilawebdev** [comment]
@markstory Good Job! Only one minor thing that needs to be fixed:

the border-bottom-color doesn't look correct. Please see:

image

**Review by billyvg — APPROVED** [review]

**Pull request merged**